### PR TITLE
DEX-738 Additional logs and orderBook improvements

### PR DESCRIPTION
**Pull request by vsuharnikov**
* Restored a submitted order validation agains counter order's price;
* Added logs;
* Uncommented check it TradeBalanceAndRoundingTestSuite;

### CODE REVIEW

File: dex/src/main/scala/com/wavesplatform/dex/AddressActor.scala [hunk 1]
```diff
@@ -532,8 +541,7 @@ object AddressActor {
   object Command {
     case class PlaceOrder(order: Order, isMarket: Boolean) extends OneOrderCommand {
       override lazy val toString =
-        s"PlaceOrder(${if (isMarket) "market" else "limit"},id=${order
-          .id()},s=${order.sender.toAddress},${order.assetPair},${order.orderType},p=${order.price},a=${order.amount})"
+        s"PlaceOrder(${if (isMarket) "market" else "limit"},id=${order.id()},js=${order.jsonStr})"
```
Comment (DenisSerebryansky) [inline dex/src/main/scala/com/wavesplatform/dex/AddressActor.scala:544]: :+1:

**Comment by vsuharnikov** [comment]
> What about solve [this](https://github.com/wavesplatform/matcher/blob/master/dex-it/src/test/scala/com/wavesplatform/it/sync/TradeBalanceAndRoundingTestSuite.scala#L286) todo?

Sure ! :) Looks like it works on my branch

**Comment by DenisSerebryansky** [comment]
Have you checked fixed test for submitted orders v3?

**Comment by vsuharnikov** [comment]
> Have you checked fixed test for submitted orders v3?
This shouldn't work. This fix isn't able to solve this issue :(

**Pull request merged**